Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -869,7 +869,7 @@ DECLARE_SOA_TABLE_FULL(StoredMcParticles_000, "McParticles", "AOD", "MCPARTICLE"
                        mcparticle::GetProcess<mcparticle::Flags, mcparticle::StatusCode>,
                        mcparticle::IsPhysicalPrimary<mcparticle::Flags>);
 
-DECLARE_SOA_TABLE_FULL(StoredMcParticles_001, "McParticles_001", "AOD", "MCPARTICLE_001", //! MC particle table, version 001
+DECLARE_SOA_TABLE_FULL(StoredMcParticles_001, "McParticles_001", "AOD", "MCPARTICLE_001", 1, //! MC particle table, version 001
```

Comment:
Thanks for this PR, @aalkin !
Could we drop in the table name here the _001 (-> "MCPARTICLE") and generate that somewhere from the version number 1 internally when the table is read from the file?

Reply:
The next step would be adjusting the reader, indeed.